packages/calcite-components/src/components/card-group/card-group.tsx

```diff
@@ -0,0 +1,251 @@
+import {
+  Component,
+  h,
+  VNode,
+  Prop,
+  Element,
+  Listen,
+  EventEmitter,
+  Event,
+  Method,
+  Watch,
+} from "@stencil/core";
+import { focusElementInGroup, toAriaBoolean } from "../../utils/dom";
+import {
+  connectInteractive,
+  disconnectInteractive,
+  InteractiveComponent,
+  InteractiveContainer,
+  updateHostInteraction,
+} from "../../utils/interactive";
+import { createObserver } from "../../utils/observers";
+import { Scale, SelectionMode } from "../interfaces";
+import { componentLoaded, setComponentLoaded, setUpLoadableComponent } from "../../utils/loadable";
+/**
+ * @slot - A slot for adding one or more `calcite-card`s.
+ */
+@Component({
+  tag: "calcite-card-group",
+  styleUrl: "card-group.scss",
+  shadow: true,
+})
+export class CardGroup implements InteractiveComponent {
+  //--------------------------------------------------------------------------
+  //
+  //  Element
+  //
+  //--------------------------------------------------------------------------
+
+  @Element() el: HTMLCalciteCardGroupElement;
+
+  //--------------------------------------------------------------------------
+  //
+  //  Public Properties
+  //
+  //--------------------------------------------------------------------------
+
+  /** When `true`, interaction is prevented and the component is displayed with lower opacity. */
+  @Prop({ reflect: true }) disabled = false;
+
+  /** Accessible name for the component. */
+  @Prop() label!: string;
+
+  /** Specifies the size of the component. Child `calcite-card`s inherit the component's value. */
+  @Prop({ reflect: true }) scale: Scale = "m";
+
+  /** Specifies the selection mode of the component. */
+  @Prop({ reflect: true }) selectionMode: Extract<
+    "multiple" | "single" | "single-persist" | "none",
+    SelectionMode
+  > = "none";
+
+  @Watch("selectionMode")
+  onSelectionModeChange(): void {
+    this.updateItems();
+  }
+
+  /**
+   * Specifies the component's selected items.
+   *
+   * @readonly
+   */
+  @Prop({ mutable: true }) selectedItems: HTMLCalciteCardElement[] = [];
+
+  //--------------------------------------------------------------------------
+  //
+  //  Private Properties
+  //
+  //--------------------------------------------------------------------------
+
+  mutationObserver = createObserver("mutation", () => this.updateItems());
```

Comment:
Do you need the mutation observer? is the slot change event not enough? Like some nested elements are being added?

Reply:
Yep, good catch. Will update.